api/src/main/java/org/openmrs/util/MemoryLeakUtil.java

```diff
@@ -99,19 +96,19 @@ public static void shutdownKeepAliveTimer() {
 	 * Clears the hibernate session factories cached in the SessionFactoryObjectFactory
 	 */
 	public static void clearHibernateSessionFactories() {
-		try {

```

Comment:
Did you test that this is no longer necessary? If yes, then we can get rid of the code instead of commenting it out